README.rst

```diff
@@ -32,10 +32,19 @@ Run Tests
 
     $ make test                                                   # to run all tests
     $ PYTEST_LOGCONF=debug-logconf.yml python tests/test_demo.py  # to run a specific test with specific logging
+    $ make test-coverage                                          # to run all tests and check the test coverage
 
 
-Check the code-style and syntax (flake8)
+Check the code-style and syntax (flake8, black, isort)
```

Comment:
Check **and fix** ...

if we include black and isort, we are doing more then just checking?

Reply:
https://github.com/vliz-be-opsci/pykg2tbl/blob/0b8d83f6ea8ab566cbf84fa32f723b5984388764/Makefile#L37

The black and isort are here with the `--check --diff` flags, meaning they will just tell you if they would change something.

To run the fix there is the `lint-fix`